bindings/python/cmds.py

```diff
@@ -0,0 +1,177 @@
+import argparse
+import sys
+from collections import Counter, defaultdict
+
+from . import REPOSITORY, ResponseOptions, Service, ServiceConfig, VectorString
+
+CMDS = {}
+
+
+def _register_cmd(cmd: str):
+    """
+    Convenience decorator function, which populates the dictionary above with
+    commands created in a declarative fashion.
+    """
+
+    def __inner(cls):
+        CMDS[cmd] = cls
+        return cls
+
+    return __inner
+
+
+@_register_cmd("translate")
+class Translate:
+    @staticmethod
+    def embed_subparser(key: str, subparsers: argparse._SubParsersAction):
+        translate = subparsers.add_parser(
+            key,
+            description="translate using a given model. Multiple models mean pivoting",
+        )
+
+        translate.add_argument(
+            "-m",
+            "--model",
+            type=str,
+            nargs="+",
+            help="Path to model file(s) to use in forward or pivot translation",
+            required=True,
+        )
+
+        translate.add_argument(
+            "-r",
+            "--repository",
+            type=str,
+            help="Repository to download model from",
+            choices=REPOSITORY.available(),
+            default="browsermt",
+        )
+
+        translate.add_argument(
+            "--num-workers",
+            type=int,
+            help="Number of worker threads to use to translate",
+            default=4,
+        )
+
+        translate.add_argument(
+            "--log-level",
+            type=str,
+            default="off",
+            help="Set verbosity level of logging: trace, debug, info, warn, err(or), critical, off",
+        )
+
+        # Tweak response-options for quick HTML in out via commandline
+        options = translate.add_argument_group("response-options")
+        options.add_argument("--html", type=bool, default=False)
+        options.add_argument("--alignment", type=bool, default=False)
+        options.add_argument("--quality-scores", type=bool, default=False)
```

Comment:
Do these options change anything in the output? Or are they just there to test those code paths inside bergamot-translator?

Reply:
Resolving claiming this will eventually turn out to be useful.